### PR TITLE
refactor: remove now post process

### PR DESCRIPTION
**Pull request by makram93**
Moved code from another PR to here.

### CODE REVIEW

File: now_common/abstract_executors/NOWBaseIndexer/base_indexer.py [hunk 1]
```diff
@@ -13,6 +13,8 @@
 )
 from now_common.abstract_executors.NOWBaseIndexer.ranking import merge_matches_sum
 
+CLOUD_BUCKET_PRE_FIXES = ['s3://']
```
Comment (violenil) [inline now_common/abstract_executors/NOWBaseIndexer/base_indexer.py:16]:
```suggestion
CLOUD_BUCKET_PREFIXES = ['s3://']
```

File: tests/now_executors/NOWPostprocessor/test_integration.py [hunk 2]
```diff
@@ -1,36 +0,0 @@
-from copy import deepcopy
-
-from docarray import Document, DocumentArray
-from jina import Flow
-from now_executors.NOWPostprocessor.executor import NOWPostprocessor
-
-
-def test_postprocessing():
```
Comment (joschkabraun) [inline tests/now_executors/NOWPostprocessor/test_integration.py:8]: I would add those tests to the executor tests of the base indexer

**Review by joschkabraun — CHANGES_REQUESTED** [review]
Nice refactoring!
Would only add test for it

File: tests/now_executors/NOWBaseIndexer/test_exec.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                Document(blob=b'b12', uri='file_will.never_exist'),
+                doc_blob,
+                doc_tens,
+            ]
+        )
+
+        f.post('/index', deepcopy(inputs), parameters={})
+
+        response = f.post('/test_list')
+
```
Comment (makram93) [inline tests/now_executors/NOWBaseIndexer/test_exec.py:69]: Test it out with search() as well

**Review by florian-hoenicke — APPROVED** [review]
LGTM can be merged

**Pull request merged**